Muvr/MRAppDelegate.swift

```diff
@@ -521,29 +527,41 @@ class MRAppDelegate: UIResponder, UIApplicationDelegate, CLLocationManagerDelega
     ///
     /// the list of predefined exercise plans
     ///
-    private var predefinedPlans: [MKExercisePlan] {
+    var predefinedSessions: [MRSessionType] {
         let bundlePath = NSBundle(forClass: MRAppDelegate.self).pathForResource("Sessions", ofType: "bundle")!
         let bundle = NSBundle(path: bundlePath)!
-        return bundle.pathsForResourcesOfType("json", inDirectory: nil).flatMap { MKExercisePlan(json: NSData(contentsOfFile: $0)!) }
+        return bundle.pathsForResourcesOfType("json", inDirectory: nil).flatMap {
+            MKExercisePlan(json: NSData(contentsOfFile: $0)!).map { .Predefined(plan: $0) }
+        }
+    }
+    
+    ///
+    /// The default exercise plan
+    ///
+    private var defaultSession: MRSessionType? {
```

Comment:
Let's be strict about our naming: ``defaultSessionType``. (Also the same comment above, ``predefinedSessionTypes``.)